<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Interface</class>
 <widget class="QWidget" name="Interface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1098</width>
    <height>712</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1131</width>
     <height>711</height>
    </rect>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideNone</enum>
   </property>
   <widget class="QWidget" name="image">
    <attribute name="title">
     <string>图片检测</string>
    </attribute>
    <widget class="QLabel" name="ini_img">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>80</y>
       <width>400</width>
       <height>400</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="pro_img">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>80</y>
       <width>400</width>
       <height>400</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="img_up">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>560</y>
       <width>401</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>上传图片</string>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="img_dec">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>560</y>
       <width>401</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>开始检测</string>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="video">
    <attribute name="title">
     <string>视频检测</string>
    </attribute>
    <widget class="QLabel" name="vid">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>50</y>
       <width>480</width>
       <height>480</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="vid_up">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>560</y>
       <width>401</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>上传视频</string>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="vid_dec">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>560</y>
       <width>401</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>开始检测</string>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="camara">
    <attribute name="title">
     <string>摄像头检测</string>
    </attribute>
    <widget class="QLabel" name="cam">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>50</y>
       <width>480</width>
       <height>480</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="cam_on">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>560</y>
       <width>401</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>打开摄像头</string>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="cam_off">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>560</y>
       <width>401</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>停止检测</string>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
